CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
  - Sticker methods to PartialGuild.
  - Helpful methods to Application object.
+ - Missing Audit Log change keys.
+ - Sticker Audit Log event types.
 
 ### Changed
  - Fix errors in rest sticker method docstrings.
+
+### Fixed
+ - Handling of interaction models passed to the webhook message endpoints as the "webhook" field.
```

Comment:
Just clarification, this was done as part of https://github.com/hikari-py/hikari/commit/795e56e599e6232f776f8c0c2aefaec280b42d9f, but it was pushed to master by mistake, so it is added here